osgar/lib/quaternion.py

```diff
@@ -110,5 +110,58 @@ def angle_between(quaternion0, quaternion1):
     val = 2 * inner_product**2 - 1
     return math.acos(max(-1.0, min(1.0, val)))
 
+def slerp(A, B, t):
+    """
+        Interpolates two quaternions utilizing the Spherical Linear
+            intERPolation (SLERP).
+
+        The input quaternions are given as list of four float values.
+        The order of their coordinates is not important, hence, a quaternion
```

Comment:
well, maybe in this function the order is not important but I would not suggest to use different one to what is in the "header" of this file